rules/limit_tokens.test.js

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2023 DigitalOcean
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+'use strict';
+
+/**
+ *
+ * @typedef {import('markdown-it/lib/token').Token} token
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+const mdEmpty = require('markdown-it')().use(require('..'), {
```

Comment:
Instead of importing the whole repo, I would prefer our tests are isolated to this plugin specifically, please

Reply:
Updated.